jib-core/src/main/java/com/google/cloud/tools/jib/frontend/BuildStepsRunner.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google LLC. All rights reserved.
+ * Copyright 2018 Google LLC. All Rights Reserved.
```

Comment:
Looks like some capitalization happened here.

Reply:
It looks like this happens in a few places in the project, not just this PR. I'll fix the rest in a follow-up.